src/box/iproto.cc

```diff
@@ -988,6 +1000,27 @@ iproto_connection_input_buffer(struct iproto_connection *con)
 	return new_ibuf;
 }
 
+static size_t
+iproto_thread_get_max_stream_queue_length(struct iproto_thread *iproto_thread)
```

Comment:
Please add a brief comment explaining what this function returns and how it computes the result.

Reply:
fixed